modelplugin/aether-5.0.0/modelplugin/modelplugin.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2021-present Open Networking Foundation.
```

Comment:
same with this file - not needed

Reply:
This would only affect use by the operator, right? If sdcore-adapter uses modelplugin.go, it would still need to be there?